api/routes/api/user.go

```diff
@@ -127,3 +127,34 @@ func RegisterHandle(c *gin.Context) {
 		"message": "user registered",
 	})
 }
+
+func DeleteAccountHandle(c *gin.Context) {
```

Comment:
Account/ User wording not 100% clear. Account = User? Currently, I think so. Maybe we should use a unified naming.